core/server/api/v2/utils/serializers/output/pages.js

```diff
@@ -1,21 +1,37 @@
 const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:pages');
 const url = require('./utils/url');
+const date = require('./utils/date');
+
+const mapPage = (model, frame) => {
+    const jsonModel = model.toJSON(frame.options);
+    url.forPost(model.id, jsonModel, frame.options);
+
+    if (frame.options.context.public) {
+        ['created_at', 'updated_at', 'published_at'].forEach((field) => {
```

Comment:
@kirrg001 just as discussed, simple iteration over known publicly exposed fields here, but definitely could be transformed to something more generic with iteration over all object fields and transforming it if it's a date for example.